src/Styles/Book.css

```diff
@@ -0,0 +1,150 @@
+.books {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 0.938rem;
+}
+
+.book {
+  background-color: white;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  padding: 2rem;
+}
+
+.book-info {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: 1rem;
+}
+
+.book-category {
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.875rem;
+  opacity: 0.5;
+  font-weight: bold;
+  color: #121212;
+}
+
+.book-title {
+  font-family: "Roboto Slab", serif;
+  font-weight: bold;
+  margin-top: 0.188rem;
+  font-size: 1.375rem;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.book-author {
+  font-family: "Roboto Slab", serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
```

Comment:
- [ ] Kindly use the correct margins and paddings defined in the original design to be similar to it. When I added a new book I couldn't see the rest of the book list.